### ISSUE TITLE
Feature Request : Vector or high resolution signs

### ISSUE THREAD

**Issue by Golutazem** [pyrollo/display_modpack#41]
This feature request is for
vector or high resolution graphics for the signs.
For use with the 512px texture packs, such as
https://gitlab.com/VanessaE/hdx-512/

**Comment by pyrollo** [comment]
SVG files are supposed to be present for all bitmap textures. With these, you could easily create texture for whatever resolution you want.
Sometimes the same SVG file is used for several textures.
If any SVG file is missing, please, tell me which.

**Comment by Golutazem** [comment]
Does a user need to do something to get the SVGs to be used or are they supposed to be used always?  Why are there both SVG and bitmap textures? Is it so that the user can choose and switch between them?

**Comment by pyrollo** [comment]
No. SVG texture are not intended for end-user. They are provided so texture pack designers can use them to create their own textures. Someone a little bit used to texture will be able to make 512px (or whatever) textures starting from SVG files.
512px texture wont be included into display_modpack as they are rarely used (most of servers run with small textures). They should be put in the texture pack rather.
